src/data/balance/rates.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Rates for shinies and other random properties are defined in this file.
+ * CHANCE is defined as x/65536
+ * RATE is defined as 1/x
+ */
+
+/** Encounterable properties */
+/** `64/65536 -> 1/1024` */
+export const BASE_SHINY_CHANCE = 64;
+
+/** `256/65536 -> 1/256` */
+export const BASE_HIDDEN_ABILITY_CHANCE = 256;
+
+/** Egg properties */
```

Comment:
NIT: You might be better of using the `region` feature here:

```suggestion
// #region Egg properties
```

---

This will also show up in the IDE minimap

<img width="132" alt="image" src="https://github.com/user-attachments/assets/8d7a5855-9232-4465-83c1-ba71da20e127">